content/id/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -0,0 +1,684 @@
+---
+title: Mengatur Pod untuk Menggunakan ConfigMap
+content_template: templates/task
+weight: 150
+card:
+  name: tasks
+  weight: 50
+---
+
+{{% capture overview %}}
+ConfigMap mengizinkan kamu untuk memisahkan artifak-artifak konfigurasi dari konten _image_ untuk menjaga aplikasi yang dikontenerisasi tetap portabel. Artikel ini menyediakan sekumpulan contoh penerapan yang mendemonstrasikan bagaimana cara membuat ConfigMap dan mengatur Pod menggunakan data yang disimpan di dalam ConfigMap.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+
+## Membuat ConfigMap
+Kamu dapat menggunakan `kubectl create configmap` ataupun generator ConfigMap pada `kustomization.yaml` untuk membuat sebuah ConfigMap. Perlu diingat bahwa `kubectl` mulai mendukung `kustomization.yaml` sejak versi 1.14.
+
+### Membuat ConfigMap Menggunakan kubectl create configmap
+
+Gunakan perintah `kubectl create configmap` untuk membuat ConfigMap dari [direktori](#create-configmaps-from-directories), [berkas](#create-configmaps-from-files), ataupun [nilai-nilai yang harfiah (_literal values_)](#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap <map-name> <data-source>
+```
+
+di mana \<map-name> merupakan nama yang ingin kamu berikan pada ConfigMap tersebut dan \<data-source> adalah direktori, berkas, atau nilai harfiah yang digunakan sebagai sumber data.
+Nama dari sebuah objek ConfigMap haruslah berupa
+[nama subdomain DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) yang sah.
+
+Ketika kamu membuat ConfigMap dari sebuah berkas, secara bawaan, _basename_ dari berkas tersebut akan menjadi kunci pada \<data-source>, dan isi dari berkas tersebut akan menjadi nilai dari kunci tersebut.
```

Comment:
This is what's been in my head, but I decided to go with `kunci: nilai` since `value` related words  have been translated to Bahasa Indonesia (e.g. `nilai harfiah`). Although keeping it as `key: value` is also good since people are used to it.

Reply:
discussed this with other reviewers, this is harder and longer debate, let's just stick on what you already have and revisit this term in the future

Reply:
Alright! Let's talk about it another time.